src/OSPSuite.Presentation/Presenters/Charts/CurveSettingsPresenter.cs

```diff
@@ -203,5 +208,32 @@ public void UpdateColorForCurve(Curve curve, Color color)
          var curveDTO = findCurveDTOFor(curve);
          UpdateCurveColor(curveDTO, color);
       }
+
+      public void EditProperties(List<CurveDTO> selectedCurveDTOs)
+      {
+         using (var multiEditorPresenter = _applicationController.Start<ICurveMultiItemEditorPresenter>())
+         {
+            multiEditorPresenter.Show();
+
+            if (multiEditorPresenter.Canceled())
+               return;
+
+            SelectedCurveValues selectedValues = multiEditorPresenter.GetSelectedValues();
+            foreach (var curveDTO in selectedCurveDTOs)
+            {
+               if (selectedValues.Color != null) curveDTO.Color = (Color)selectedValues.Color;
+               if (Enum.TryParse(selectedValues.Style, out LineStyles selectedStyle)) 
```

Comment:
Lol..yes this was very suspicious. 
Just set the value using real types (maybe nullable) instead or all those cast. 
Unless there is something that I don't understand? In that case I need some explanations